GalaxyZooTouchTable/GalaxyZooTouchTable/GalaxyZooTouchTable.csproj

```diff
@@ -388,9 +389,10 @@
     </None>
     <None Include="App.Release.config">
       <DependentUpon>App.config</DependentUpon>
+      <SubType>Designer</SubType>
     </None>
     <Resource Include="Fonts\Garamond.ttf" />
-    <None Include="GalaxyZooTouchTable_TemporaryKey.pfx" />
+    <None Include="GalaxyZooTouchTable_1_TemporaryKey.pfx" />
```

Comment:
This appears to change in a few different PRs. I assume we renamed something and that's propagating through the change sets?

Reply:
Yes, I couldn't build and publish any of these changes without creating a new temporary key (that lasts for a year), so you'll see these changes in any of the current PRs until they hit master. There's more information about this in the repo's wiki under the troubleshooting section.